doc/release-notes/9.3.md

```diff
@@ -0,0 +1,32 @@
+# Release notes for Gluster 9.3
+
+This is a bugfix and improvement release. The release notes for [9.0](9.0.md), [9.1](9.1.md), [9.2](9.2.md) contain a listing of all the new features that were added and bugs fixed in the GlusterFS 9 stable release.
+
+**NOTE:**
+- Next minor release tentative date: Week of 30th Aug, 2021
+- Users are highly encouraged to upgrade to newer releases of GlusterFS.
+
+## Important fixes in this release
+
+- Core dumps on Gluster 9 - 3 replicas [#2443] (https://github.com/gluster/glusterfs/issues/2443)
+- geo-rep: Improve handling of gfid mismatches [#2423] (https://github.com/gluster/glusterfs/issues/2388)
+- auth.allow list is corrupted after add-brick (buffer overflow?) [#2524] (https://github.com/gluster/glusterfs/issues/2524)
```

Comment:
Pls drop the space between [text]' '(link)
The rest looks great

Reply:
@tshacked done!